runtime/jvmti/jvmtiStartup.c

```diff
@@ -575,6 +575,12 @@ loadAgentLibraryOnAttach(struct J9JavaVM * vm, const char * library, const char
 
 	Trc_JVMTI_loadAgentLibraryOnAttach_Entry(library);
 
+	if (J9_ARE_NO_BITS_SET(vm->runtimeFlags, J9_RUNTIME_ALLOW_DYNAMIC_AGENT)){
```

Comment:
Missing space before `{`.
While you're here, please provide initial values for `rc` and `libraryLength` in their declarations above.